lib/phoenix/digester.ex

```diff
@@ -68,17 +99,23 @@ defmodule Phoenix.Digester do
   end
 
   defp map_file(file_path, input_path) do
+    {:ok, stats} = File.stat(file_path)
     %{absolute_path: file_path,
       relative_path: Path.relative_to(file_path, input_path) |> Path.dirname(),
       filename: Path.basename(file_path),
+      mtime: stats.mtime,
```

Comment:
You are correct.  Sprockets uses `Time.now`.  I've switched over the new digest mapping to use the current time but am still using the file `mtime` in the case of files added switching from the old manifest format.